data/seeds/002-profiles.js

```diff
@@ -29,6 +29,48 @@ const profiles = [
     profile_id: 'aa388259-28a9-4fa3-a988-ec08070a4f2e',
     role: 'mentee',
   },
+  // bear001
```

Comment:
@paulstgermain Can this be removed now that we are getting this information from Auth0?

Reply:
Unfortunately, no - we need this user data in our database so that we can pull the data from DS.

Under ordinary circumstances, a user will sign up and get their info (Auth0 ID included!) cataloged in our database at that same moment.

After that every time they sign in we take their ID from Auth0, check with our (WEB) database to make sure we have them on record, and then send their ID to the (DS) database to pull the rest of their information once we've confirmed they're a registered user with us.

Our test users never went through that signup process, so the easiest way to get their info into our database is through our seeds - this won't be in play in the final release, but helps us for now! :D